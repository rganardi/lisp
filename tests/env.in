(define 0 (quote zero))

0

,p

(define list
  (lambda x x))

,p

(define 0 (quote (list second zero)))

,p

(define 1 (quote a))

(define 0 (list (quote second) (quote zero)))

,p

(define 1 0)

0
1
,p
