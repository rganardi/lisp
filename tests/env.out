parse_eval: unspecified
zero
printing environment
0	->	zero
nil	->	
parse_eval: unspecified
parse_eval: unspecified
printing environment
list	->	(lambda x x)
0	->	zero
nil	->	
parse_eval: unspecified
parse_eval: unspecified
printing environment
list	->	(lambda x x)
0	->	(list second zero)
nil	->	
parse_eval: unspecified
parse_eval: unspecified
parse_eval: unspecified
printing environment
1	->	a
list	->	(lambda x x)
0	->	(second zero)
nil	->	
parse_eval: unspecified
parse_eval: unspecified
(second zero)
(second zero)
printing environment
1	->	(second zero)
list	->	(lambda x x)
0	->	(second zero)
nil	->	
parse_eval: unspecified
